app/views/admin/announcements/_form.html.slim

```diff
@@ -0,0 +1,6 @@
+= simple_form_for @announcement, resource: :admin_announcement do |f|
```

Comment:
Eh? The `announcement#new` action didn't initialize `@announcement` will this work?